### ISSUE TITLE
BL7 upgrade: Update color styling of "secondary" buttons

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/exhibits#1582]
We currently (in -stage) have two types of secondary buttons in use:

- **A:**  .`btn-secondary-outline` (search result control buttons, search constraint pills)
- **B:** `.btn-secondary` (example: "Save this search" button, Configuration > Metadata page)

Example of how those look now:

<img width="1147" alt="Screen Shot 2020-01-30 at 3 34 12 PM" src="https://user-images.githubusercontent.com/101482/73496567-1d009200-4376-11ea-9017-39976bf9c204.png">

I don't think we want to use the outline style at all in Exhibits, so I think we can style these two variations of `btn-secondary` in the same way.

In the specifications below, unless otherwise noted, the colors **are not** currently defined with variables in the `bl7` branch, but **are** what we currently use in -prod.

Ideally, I would like to switch more of these to our `sul_variables` colors, but it feels like a big task to both get things back to looking good enough for production _and_ figure out a new color scheme, so I'd like to save big decisions on the latter until after we have everything else looking good.

### Normal state
- [ ] Background color: `#f2f1eb` - currently used for these buttons in production
- [ ] Border color: `#dad7cb` - different from -prod, but `$cloud` in the `bl7` branch `sul_variables.scss` file

### Hover state
- [ ] Background color: `#dad7cb` defined as `$sul-btn-secondary-color` in the `bl7` branch `sul_variables.scss` file
- [ ] Border color: `#b6b1a9`

### Active state
- [ ] Background color: `#d0ccb6`
- [ ] Border color: `#8c8c8c`
- [ ] box-shadow: `inset 0 3px 5px rgba(0,0,0,0.125)`

Here's how the previous screenshot looks with those three state styles applied:

<img width="1147" alt="Screen Shot 2020-01-30 at 4 01 30 PM" src="https://user-images.githubusercontent.com/101482/73498506-bdf14c00-437a-11ea-96b9-c0b9cc82f286.png">

---

- [ ] Note that the text and svg colors of these secondary buttons currently (in -prod) change to white in the hover and active states. Given the color choices above, we don't want these colors to change to white in any state. They should continue to use the default text/svg color that is used in the normal state.

**Comment by cbeer** [comment]
The bootstrap button mixin, by default, calculates the hover and active styles using the normal state. Is it worth trying their default behaviors before explicitly specifying each state?

backgrounds: #f2f1eb, hover #f2f1eb, active #dedbcc
border: #dad7cb  hover #c5c0ad active #c1bba7

Or, despite the concerns about changing colors, maybe we should try `sul-btn-secondary-color / cloud`? I believe that's what mylibrary is currently using.

**Comment by ggeisler** [comment]
So we're currently using the BS button mixin to calculate the colors? That might explain why I couldn't find a hard-coded specification for some of the colors I was seeing in the rendered display.

Knowing that, I'm fine with variations of the colors I specified if they can be automatically calculated by the button mixin. I do like the idea of not having to hardcode the, and in some cases they appear close to what I specified, so why not. But not everything seems like it will work. 

> backgrounds: #f2f1eb, hover #f2f1eb, active #dedbcc
border: #dad7cb hover #c5c0ad active #c1bba7

Normal state: both bg and border colors same as I suggested so, yes, I like those

Hover state background-color: same as the normal state. Not sure if that is a typo or what the mixin does, but I don't think that'll work. The border change on hover feels much too subtle to rely on alone for conveying the hover state.

Hover state border-color: close to what I suggested, and seems fine

Active state background-color: sure, that looks pretty close to what I had specified

Active state border-color: also looks fine

So overall, the mixin-produced values you list all look fine to me, except the hover background-color, which needs to changed to be somewhat darker. I'm not sure the MyLibrary hover color will work as well here, because in MyLibrary the normal state background color is lighter, so the difference in contrast won't be as noticeable unless we switch the normal state to use the same color as MyLibrary. I tried that, and it didn't look right in Exhibits; it clashes with the background color of the facet panels, so we'd have to change those colors as well. Overall, while I completely agree it would be good to find more commonality across our applications, MyLibrary feels like a very different application (mobile-oriented, not a lot of controls on a single page, etc.) and I'm wary of just adopting styling from it without doing a complete visual design overhaul of Exhibits.
